demo/tests/test_end_point.py

```diff
@@ -915,11 +915,9 @@ def test_create_form_conditions_invalid_dup(self):
         data['fields'] = copy.deepcopy(self.fields_with_validation)
         data['conditions'] = copy.deepcopy(self.valid_conditions_invalid_dup)
         serializer = FormidableSerializer(data=data)
-        self.assertFalse(serializer.is_valid())
-        self.assertEqual(
-            serializer.errors['non_field_errors'][0],
-            'Action display_iff in condition (my condition) is used many times for the same fields (input-date)' # noqa
-        )
+        # TODO decide if the validation should fail
```

Comment:
not sure if I want to keep this TODO here

Reply:
BTW, why an action name is `display_iff` and not just a `display_if`?

Reply:
I was unsure about this one: we have the exact same condition two times. Should it be allowed? Either for displaying or making mandatory it does't impact the form so I would say yes.

Then this test should be renamed with "valid" instead of "invalid", and the "valid_conditions_invalid_dup" in "valid_conditions_with_dup".

Reply:
> BTW, why an action name is display_iff and not just a display_if?

I don't know. I think we may ask @mgu

Reply:
ok, so I'll let this "TODO" and we'll work on it in an other task. @syldlb could you open a github issue in this repository, please?

Reply:
https://en.wikipedia.org/wiki/iff

usual mathematical abbreviation

Reply:
but as you are changing the way it works, it makes no sense to keep this name…

Reply:
I see... it'd be cool to make a data migration using the JSON migrations, but not here yet ; changing the name of the only action available is not worth it IMO.